app/forms/publish/user_form.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Publish
+  class UserForm < Form
+    FIELDS = %i[
+      first_name
+      last_name
+      email
+      id
+      code
+      authenticity_token
+    ].freeze
+
+    attr_accessor(*FIELDS)
+
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :email, presence: true, format: { with: /\A.*@.*\z/, message: "Enter an email address in the correct format, like name@example.com" }
```

Comment:
Cna we move the validation error message to translation please

Reply:
Done here: https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/2aa2b039f2c6838b2775a0789713ea26c4f742a1